pages/search/search.jsx

```diff
@@ -31,11 +31,17 @@ class Search extends React.Component {
   }
 
   getSearchCriteria(props) {
-    return {
-      keywordSearched: props.location.query.keyword,
-      moderationState: { value: ``, label: props.location.query.moderationstate || DEFAULT_MODERATION_FILTER },
-      featured: props.location.query.featured || ``
+    let criteria = {
+      keywordSearched: props.location.query.keyword
     };
+
+    if (this.props.moderation) {
+      // the following states are only useful on moderation mode
+      if (props.location.query.featured === `True`) criteria.featured = props.location.query.featured;
```

Comment:
always better to use `{}` so that it's super obvious what is, and is not, controlled by the `if` statement. Even if it's a one liner